src/auth/auth.service.ts

```diff
@@ -60,6 +61,32 @@ export class AuthService {
     await this.usersRepository.verifyUser(user.id);
   }
 
+  async sendResetPasswordEmail(email: string): Promise<void> {
+    const user = await this.usersRepository.getUserByEmail(email);
+
+    if (!user) {
+      return;
+    }
+
+    const payload: JwtPayload = { userId: user.id, email };
+    const resetPasswordToken = await this.jwtService.signAsync(payload, {
+      expiresIn: '1d',
+      secret: this.configService.get<string>('JWT_SECRET'),
```

Comment:
is this the same JWT secret key as for authentication? I think it's more secure to add a separate private key, e.g. `CONFIRM_ACCOUNT_JWT_SECRET`